airbyte-integrations/connectors/source-exchange-rates/integration_tests/sample_state.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Do we need empty files `integration_tests/configured_catalog.json` and `integration_tests/sample_config.json`?

Reply:
These are not empty. They were just renamed.